ts-tests/scripts/start-relay-and-para-chain.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# configure the relative paths of binary
+PARACHAIN_ID=2022
+GIT_ROOT=`git rev-parse --show-toplevel`
+LITENTRY_BIN=$GIT_ROOT/target/release/litentry-collator
+POLKADOT_BIN=$GIT_ROOT/polkadot/target/release/polkadot
+
+# generate chain spec
+ROCOCO_CHAINSPEC=rococo-local-chain-spec.json
+$POLKADOT_BIN build-spec --chain rococo-local --disable-default-bootnode --raw > $ROCOCO_CHAINSPEC
+
+# generate genesis head and wasm validation files
+$LITENTRY_BIN export-genesis-state --parachain-id $PARACHAIN_ID > para-$PARACHAIN_ID-genesis
+$LITENTRY_BIN export-genesis-wasm > para-$PARACHAIN_ID-wasm
+
+# run alice and bob as relay nodes
+$POLKADOT_BIN --chain $ROCOCO_CHAINSPEC --alice --tmp --port 30333 --ws-port 9944 &> "$1/relay.alice.log" &
+sleep 3
+$POLKADOT_BIN --chain $ROCOCO_CHAINSPEC --bob --tmp --port 30334 --ws-port 9945  &> "$1/relay.bob.log" &
```

Comment:
I'd like to alias a name for `$1`

e.g. 
```
tmp_prefix=$1
```
then
```
$tmp_prefix/relay.alice.log
```

It's much easier for us to understand the source code.

Reply:
sure it makes sense. I'll update it